### PR TITLE
Gui

### PR DESCRIPTION
**Pull request by Lia4alNoor**

### CODE REVIEW

File: src/ninemensmorris/BoardComp.java [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+        setHeight(SaMPLENMM.POS_SIZE);
+        setWidth(SaMPLENMM.POS_SIZE);
+        relocate(x*SaMPLENMM.POS_SIZE,y*SaMPLENMM.POS_SIZE);
+        setFill(posMove ? Color.GREEN: Color.GREY);
+    }
+
+    public BoardComp() {
+    }
+    
+      public void PlaceCoin(Pane root,CoinType ct,double [] xy){
```
Comment (lillieMaiBauer) [inline src/ninemensmorris/BoardComp.java]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

File: src/ninemensmorris/NMMLogic.java [hunk 2]
```diff
@@ -74,7 +74,7 @@
                         new InputStreamReader(
                                 getClass().getResourceAsStream("/ninemensmorris/resources/slot_index_ref.json")));
```
Comment (aidanBurton) [inline src/ninemensmorris/NMMLogic.java]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [UI: Usage of GetResource may be unsafe if class is extended (UI_INHERITANCE_UNSAFE_GETRESOURCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ui-usage-of-getresource-may-be-unsafe-if-class-is-extended-ui-inheritance-unsafe-getresource).
Calling this.getClass().getResource(...) could give results other than expected if this class is extended by a class in another package.

Comment (EltoCode) [reply src/ninemensmorris/NMMLogic.java]: This issue was fixed in a newer version, Thank ya kindly, By Giggity

Comment (aidanBurton) [reply src/ninemensmorris/NMMLogic.java]: Thanks for your reply. Happy to help :)

**Comment by EltoCode** [comment]
This pull request was opened by error

**Pull request closed without merging**